### PR TITLE
Gutenboarding: update plans grid header

### LINKED ISSUE #40822: Gutenboarding: upgrade plans grid visuals and copy
This is how plans grid currently looks like in Gutenboarding before launching the site, or if the user chooses a paid domain:

image

### Copy when purchasing domain

@johnHackworth [suggests](https://github.com/Automattic/wp-calypso/pull/40813#issuecomment-609876812) this copy:

> Pick one of our paid plans to get [domain] for your site!
> You can change to any other plans later, and upgrade as you grow.
>
> Not sure yet? start with the _free plan_ and select a paid domain later!

In Figma we have this:

> Choose a plan
> Pick a plan that's right for you. Switch plans as you needs change. There's no risk, you can cancel for a full refund within 30 days.

### Copy when launching

> Choose a plan
> Pick a plan that's right for you. Switch plans as you needs change. There's no risk, you can cancel for a full refund within 30 days.

## Styles

Eventually, we'll want it to look like this:

image

For this calypso grid we don't want to do very heavy modifications, so let's stick to very basics. I'm not sure if these modifications I'm suggesting are "too much" CSS overrides and would just render the whole thing too unstable. @johnHackworth what's the timeline when we'd work on a custom grid? We might also just decide to replace the header with our custom header and leave the grid CSS without overrides. I don't think using a completely white background with the current grid works without further CSS overrides.

Just pseudo-code to give direction:

```css 
body {
   background: #fff;
}

.plan-features__header {
  border-color: transparent;
}

tr:last-child .plan-features__table-item,
tr:first-child .plan-features__table-item {
  border-radius: 0;
}

.plan-features__header,
.plan-features__table th,
.plan-features__table td {
  background: rgba(248, 248, 248, 0.7);
}

.plan-price__integer {
  font-weight: 500;
}

.plan-price__integer {
  font-weight: 500;
}

.plan-features__header-title {
  font-weight: 500;
}

.wordpress-logo { 
  display: none;
}
```

For the heading section:

- Use Recoletta font for the 1st heading.
- Left-align text
- Remove W logo
- Right align "free plan"
- Generally aim to match header in Figma. Can we replace the header completely, instead of doing CSS overrides?

We would end up with something like this:

image

@olaolusoga @dubielzyk @johnHackworth thoughts?

### PR DESCRIPTION
**Pull request by razvanpapadopol**
#### Changes proposed in this Pull Request

*

#### Testing instructions

*

Fixes #40822

### CODE REVIEW

File: client/my-sites/plan-features/scroller.jsx [hunk 1]
```diff
@@ -123,6 +123,7 @@ export default class PlanFeaturesScroller extends PureComponent {
 	}
 
 	handleWindowResize = () => {
+		/* global cancelAnimationFrame */
```
Comment (simison) [inline client/my-sites/plan-features/scroller.jsx:126]: `window.cancelAnimationFrame` should do the trick 👍

**Comment by simison** [comment]
Adding top bar is likely too complicated for this view that it's not worth it.

**Comment by johnHackworth** [comment]
> Can we add the same padding as we have on the other screens?

@dubielzyk could you give me some values? I've been playing by hearing here, but it's not easy because the text sizes of the header are different from other parts of gutenboarding

**Comment by dubielzyk** [comment]
>@dubielzyk could you give me some values? 

My bad, @johnHackworth.  [This is the SASS variable](https://github.com/Automattic/wp-calypso/blob/master/client/landing/gutenboarding/onboarding-block/design-selector/style.scss#L13) and those values translate to `margin: 64px 0 80px`

**Comment by johnHackworth** [comment]
@dubielzyk  ok, updated to look like this:

image

(I hope this is what you meant :D )

**Comment by simison** [comment]
> ok, updated to look like this

@dubielzyk @johnHackworth heh, I was faster (https://github.com/Automattic/wp-calypso/pull/40860/commits/65cca0bb75c5f6e345dd1a540515b60fca153fff). ;-)


Screenshot 2020-04-07 at 22 36 36

**Comment by johnHackworth** [comment]
let's merge this :ship:  

Any further feedback will be addressed in other PRs, Jon

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/3184404

Thank you @razvanpapadopol for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/3184404).

**Pull request merged**